### ISSUE TITLE
Copy UnityFramework.framework: No such file or directory [IOS]

### ISSUE THREAD

**Issue by VitamineC** [azesmway/react-native-unity#11]
Hi
I try to integrate the unity project using this library and get Copy UnityFramework.framework: No such file or directory. 

IOS 15, Xcode 13, emulator.

**Comment by azesmway** [comment]
> Copy UnityFramework.framework: No such file or directory.

Hi
I don't understand - why copy it?

**Comment by VitamineC** [comment]
I got that error while trying to build the app. 
On real device all works

**Comment by azesmway** [comment]
yes, this component only works on a real device. I didn't even try to make it work in emulators)

**Comment by VitamineC** [comment]
probably we should add this information to readme?)
I lost a few hours to try start it on emulator

**Comment by azesmway** [comment]
Ok I'll add!

but I think that you can still run it - I just didn’t have such a task
